identity/src/routes/ministry/associate_users.rs

```diff
@@ -55,6 +55,8 @@ impl crate::routes::Routes {
                 .push_bind(&*id)
                 .push_bind(&user.role_id);
         })
+		.push("ON CONFLICT (user_id, ministry_id) DO UPDATE
```

Comment:
You'll need a whitespace before the `ON` word else it will result in invalid SQL syntax